### DISCUSSION TITLE
Microsoft edge chat gpt

### DISCUSSION THREAD

**Discussion by DELINARD** [community/community#63566] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Suite au case 1055841856
Edge 115.0.1901.196

Microsoft n'a donnée aucune réponse pour le moment. Suite au mail de Google apparemment oui le problème à été pris en charge il y a moins d'activité sur réseaux Android.

JUG-841-21111 

Franck DELINARD pour l'équipe support ou expertise développement et partenaires Microsoft. 

Bonjour pourriez vous m'expliquer dans la surveillance réseau de mon smartphone android samsung s23 ultra j'ai une api qui est reconnu inconnu. Toutes les autres api sont bien reconnu avec l'icône edge. 

Api: sapphire.api.microsoftapp.net

![Screenshot_20230808_121123_PCAPdroid](https://github.com/community/community/assets/64479958/a76f756e-003f-469b-95b1-ff0555ea6fea)

**Comment by queenofcorgis** [comment]
Hi @DELINARD,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Samsung or Microsoft. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
